api/renter.go

```diff
@@ -29,7 +31,6 @@ type FileInfo struct {
 	UploadProgress float32
```

Comment:
I guess I'm just surprised it's not an int. Like bytes or segments vs. full size.